VERSION.txt

```diff
@@ -1 +1 @@
-8da7f9d.dirty
\ No newline at end of file
+0.0.3-1-g68a36e6.dirty
```

Comment:
pls remove this file from git, this must not be tracked, since the version is derived from git